public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -44,26 +43,11 @@ interface OwnProps {
   sourcePanel: PanelModel;
 }
 
-interface ConnectedProps {
-  location: LocationState;
-  plugin?: PanelPlugin;
-  panel: PanelModel;
-  initDone: boolean;
-  tabs: PanelEditorTab[];
-  uiState: PanelEditorUIState;
-  variables: VariableModel[];
-}
+const connector = connect(mapStateToProps, mapDispatchToProps);
 
-interface DispatchProps {
-  updateLocation: typeof updateLocation;
-  initPanelEditor: typeof initPanelEditor;
-  panelEditorCleanUp: typeof panelEditorCleanUp;
-  setDiscardChanges: typeof setDiscardChanges;
-  updatePanelEditorUIState: typeof updatePanelEditorUIState;
-  updateTimeZoneForSession: typeof updateTimeZoneForSession;
-}
+type PropsFromRedux = ConnectedProps<typeof connector>;
 
-type Props = OwnProps & ConnectedProps & DispatchProps;
+type Props = OwnProps & PropsFromRedux;
```

Comment:
```suggestion
type Props = OwnProps & ConnectedProps<typeof connector>;
```

PropsFromRedux type seems unnecessary as it's only used there